wraps/python/src/indicators/Qoute.py

```diff
@@ -0,0 +1,11 @@
+from Skender.Stock.Indicators import Quote
+from cstypes import DateTime, Decimal
+
+class Qoute(Quote):
```

Comment:
I realize you're probably spelling this differently to avoid naming conflicts, but `Qoute` it looks like a typo.  I noticed you were using `Cs` as a prefix in the `System` types, maybe we can do something like that here to make it look more intentional?

Reply:
Maybe `... import Quote as CsQuote` then keep `Quote` as the native Python class name?

Reply:
Okay. That looks better for code consistency!
```suggestion
class Qoute(CsQuote):
```